frontend/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@kubeflow/frontend": "github:Bobgy/frontend#36d6d1",
+    "@kubeflow/frontend": "github:kubeflow/frontend#209f513",
```

Comment:
https://github.com/kubeflow/frontend/commit/9023682193aafdf717f63383653552d3292d7188

Can you use this instead? My fork included the two new changes.

Reply:
Let's try to use a new hash from upstream rather than hash commits on user's forks. Are your fork's changes merged?